src/exercises/day-3/simple-gui.rs

```diff
@@ -43,14 +43,14 @@ impl Label {
 
 pub struct Button {
     label: Label,
-    callback: Box<dyn FnMut()>,
+    // callback: Box<dyn FnMut()>, // Would also take a callback
```

Comment:
Thanks! Please remove the comment too to make the exercise more focused for the students. It's already helpful that you mention the possibility for a callback in the text above.